UI/src/view/linear_asset/laneModellingForm.js

```diff
@@ -98,8 +100,10 @@
 
   var roadAddressFormStructure = {
     fields : [
-      {label: 'Osa', required: 'required', type: 'number', publicId: "endRoadPartNumber", weight: 7},
-      {label: 'Etäisyys', required: 'required', type: 'number', publicId: "endDistance", weight: 8}
+      {label: 'Osa', required: 'required', type: 'number', publicId: "startRoadPartNumber", weight: 7},
+      {label: 'Etäisyys', required: 'required', type: 'number', publicId: "startDistance", weight: 8},
+      {label: 'Osa', required: 'required', type: 'number', publicId: "endRoadPartNumber", weight: 9},
+      {label: 'Etäisyys', required: 'required', type: 'number', publicId: "endDistance", weight: 10}
```

Comment:
I think maybe it's better to have different labels names in this case.

Reply:
We use the labels for the name of the fields in the UI so I don't think we should have different labels. The publicId is different

Reply:
Talked with the team about this and is clearer why this must be like this